Source/OIDAuthorizationService.m

```diff
@@ -171,6 +171,14 @@ - (void)didFinishWithResponse:(nullable OIDAuthorizationResponse *)response
   }
 }
 
+- (void)failAuthorizationFlowWithError:(NSError *)error {
+  [self failAuthorizationFlowWithError:error];
```

Comment:
This method is calling itself right now, causing an infinite loop. Assuming it is there for backwards compatibility with the old protocol it should call the new `[self failExternalUserAgentFlowWithError:error]` instead.

Reply:
@WilliamDenniss @luksfarris Please let us know if this will be reviewed and merged. Also can you confirm on the invoking user agent flow for RP Initiated logout - https://github.com/openid/AppAuth-iOS/pull/191#issuecomment-367402450

Reply:
@dgommers I had missed that. Just fixed and rebased
@jaishankar I'll reply your comment on issue #191 soon